modules/dynamic-plugins/ref-community-plugins.adoc

```diff
@@ -10,9 +10,17 @@
 Details on how {company-name} provides support for bundled community dynamic plugins are available on the https://access.redhat.com/policy/developerhub-support-policy[Red Hat Developer Support Policy] page.
 ====
 
-{product-very-short} includes the following 0 community plugins:
+{product-very-short} includes the following 2 community plugins:
 
 [%header,cols=4*]
 |===
 |*Name* |*Plugin* |*Version* |*Path and required variables*
+|Argo CD  |`https://npmjs.com/package/@roadiehq/backstage-plugin-argo-cd/v/2.8.4[@roadiehq/backstage-plugin-argo-cd]` |2.8.4 
+|`./dynamic-plugins/dist/roadiehq-backstage-plugin-argo-cd`
```

Comment:
this plugin is now deprecated. https://github.com/redhat-developer/rhdh/pull/2375/files#diff-eca0eb2bcac439517b5ecc46e22ebdb2ed8e035dcf08062c72d2609a8462af34

So... I guess it makes sense to list it downgraded from "Supported" to "Community" but we also need a RN item for this. 

See RN text in https://issues.redhat.com/browse/RHIDP-2028